test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -143,6 +143,11 @@ test
     await t.expect(pageTitle).contains('Verify with your email');
     await t.expect(challengeEmailPageObject.getFormSubtitle())
       .contains('Check your email for a verification message. Click the verification button in your email or enter the code below to continue.');
+
+    // Verify links (switch authenticator link not present since there are no other authenticators available)
+    await t.expect(await challengeEmailPageObject.switchAuthenticatorLinkExists()).notOk();
```

Comment:
Probably OK - but if we modify the mocks at any time we may lose out on this test coverage if responses no longer have a single authenticator present.

For future safety - can we modify the `validOTPmock` above to remove **all but one** authenticator as a separate test?

Reply:
I don't think we change mocks that often, so I was thinking that we can rely on that mock. If we change it in the future, this test will fail, and I am hoping at that point the developer making the change will do whatever needed to keep the test coverage good. 
PS All the tests I am changing are relying on the mocks as they are today. 

Does this sound good? I can add the test too, but I think we are covered with what we have